verify/verify-flags/known-flags.txt

```diff
@@ -20,6 +20,10 @@ n-runs-count
 ping-server-address
 ping-server-bind-address
 ping-sleep-duration
+prometheus-scrape-etcd
+prometheus-scrape-kube-proxy
```

Comment:
Is this needed? You don't backport any Prometheus component?

Reply:
It's not necessary, but if someone uses `https://github.com/kubernetes/perf-tests#repository-setup` from master branch, then it breaks pushing.